.github/workflows/python-lint.yml

```diff
@@ -0,0 +1,46 @@
+name: lint PR
+on:
+  pull_request:
+    paths:
+      - .github/workflows/python-lint.yml
+      - .
```

Comment:
Do we need paths if we want to trigger it every time?

Reply:
yes, we can remove